run-k8s-external-storage-e2e.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+#
+# Download the Kubernetes test suite, extract it and run the external-storage suite.
+#
+# Requirements:
+# - KUBE_VERSION needs to be set in the environment (format: "v1.18.5")
+
+# exit on failure
+set -e
+
+[ -n "${KUBE_VERSION}" ] || { echo "KUBE_VERSION not set" ; exit 1 ; }
+
+# download and extract the tests
+curl -LO "https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/kubernetes-test-linux-amd64.tar.gz"
+tar xzf kubernetes-test-linux-amd64.tar.gz kubernetes/test/bin/ginkgo kubernetes/test/bin/e2e.test
+
+# generate the rbd-driver.yaml that describes the test features
+cat > rbd-driver.yaml << EOF
+ShortName: cephcsi-rbd-test
+StorageClass:
+  FromExistingClassName: csi-rbd-sc
+  #FromFile: examples/rbd/storageclass.yaml
+
+SnapshotClass:
+  # Must be set to enable snapshotting tests
+  FromName: true
+
+DriverInfo:
+  # Internal name of the driver, this is used as a display name in the test case and test objects
+  Name: test.rbd.csi.ceph.com
+
+  # The range of disk size supported by this driver
+  SupportedSizeRange:
+    Min: 1Gi
+    Max: 16Ti
+
+  # Map of strings for supported FS types
+  SupportedFsType:
+    ext4: {}
+    xfs: {}
+
+  # Map of strings for supported mount options
+  SupportedMountOption:
+    rw: {}
+
+  # Map of strings for required mount options
+  RequiredMountOption:
+    rw: {}
+
+  # Optional list of access modes required for provisiong. Default is RWO
+  # RequiredAcccessModes:
+
+  # Map that represents the capabilities the driver supports
+  Capabilities:
+    # Data is persistest accross pod restarts
+    persistence: true
+
+    # Volume ownership via fsGroup
+    fsGroup: false
+
+    # Raw block mode
+    block: true
+
+    # Exec a file in the volume
+    exec: true
+
+    # Support for volume limits
+    volumeLimits: false
+
+    # Support for volume expansion in controllers
```

Comment:
we do support expansion I think we need to be set to true

Reply:
Sure, but I just hope to get something that passes first. I am considering to place these `driver.yaml` files in the master branch instead of in the ci/centos one. That makes it possible to add features and have them tested with a single PR.

Reply:
Yes sounds good to add a Job and move the yaml file to the master branch.